core/build.gradle.kts

```diff
@@ -37,6 +37,7 @@ dependencies {
   testImplementation(libs.junit.jupiter.api)
   testImplementation(libs.junit.jupiter.params)
   testImplementation(libs.mockito.core)
+  testImplementation(libs.testcontainers)
```

Comment:
Why do we need to introduce this?

Reply:
I used the awaitability from `testcontainers`,  will use from `awaitility` instread